### DISCUSSION TITLE
Declarative Form from JSON

### DISCUSSION THREAD

**Discussion by dealenx** [react-hook-form/react-hook-form#7794] [Q&A] [answered]
Hello, I'm looking for declarative form generator from JSON.

Can i use `react-hook-form` for for that purpose?

For example, the concept where form schema is transforming to JSX ([link](https://github.com/dbagia/declarative-form-generator)):

![img](https://github.com/dbagia/declarative-form-generator/raw/master/aToB.PNG)

Can you guide me on how this could be applied?

Thank you in advance for any information!

**Comment by bluebill1049** [comment, marked as answer]
would this help? https://github.com/vtex/react-hook-form-jsonschema

**Reply by dealenx** [reply]
Thank you for the information!

**Reply by bluebill1049** [reply]
hopefully we can get to this PR done as well: https://github.com/react-hook-form/resolvers/pull/318 then we can have a resolver for JSON schema.
